doc/guide/feature-virtualmachines.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0"?>
+<!DOCTYPE chapter PUBLIC "-//OASIS//DTD DocBook XML V4.3//EN"
+        "http://www.oasis-open.org/docbook/xml/4.3/docbookx.dtd">
+<chapter id="feature-virtualmachines">
+    <title>Virtual Machines</title>
+
+    <para>Cockpit can manage virtual machines running on the host.
+        These can be accessed from menu via <emphasis>Virtual Machines</emphasis></para>
+
+    <para>Primary datasource is QEMU/Libvirt, access to Libvirtd is wrapped by the <emphasis>virsh</emphasis> tool.</para>
+
+    <section id="feature-virtualmachines-nestedvirtualization">
+    <title>Nested Virtualization</title>
+    <para>
+        In order to run <emphasis>cockpit-machines</emphasis> plugin inside a virtual machine
+        or run the tests for this component, you'll need nested KVM virtualization enabled
+        (refer <code>test/verify/nested-kvm</code> or
+        <ulink url="https://fedoraproject.org/wiki/How_to_enable_nested_virtualization_in_KVM">
+            How to enable nested virtualization in KVM</ulink> to get more information).
+    </para>
+
+    <para>
+        With nested virtualization enabled, Vagrant can be used:
```

Comment:
From here on down these seem like development instructions rather than admin instructions. Should this part go into HACKING.md or a pkg/machines/README.md?

Reply:
ok, moved to pkg/machines/README.md